mne/viz/_brain/tests/test_brain.py

```diff
@@ -71,114 +75,81 @@ def test_brain_init(renderer):
     with pytest.raises(KeyError):
         _Brain(subject_id=subject_id, hemi='foo', surf=surf)
 
+    # cortex
     for cortex in ['classic', 'low_contrast', 'high_contrast', 'bone']:
```

Comment:
This does 4 brain inits, `'classic'` is the default (right?) so maybe just test one of these other ones?